anvil/anvil-compiler/src/main/java/com/duckduckgo/anvil/compiler/ContributesRemoteFeatureCodeGenerator.kt

```diff
@@ -197,6 +197,24 @@ class ContributesRemoteFeatureCodeGenerator : CodeGenerator {
                                                         }
                                                     }
                                                 }
+                                                override suspend fun getAllTogglesForParent(name: String): List<Toggle> {
+                                                    return feature.javaClass.declaredMethods.mapNotNull { method ->
+                                                        if (method.genericReturnType.toString().contains(Toggle::class.java.canonicalName!!)) {
+                                                            method.invoke(feature) as Toggle
+                                                        } else {
+                                                            null
+                                                        }
+                                                    }
+                                                }
+                                                override suspend fun getAllActiveExperimentToggles(): List<Toggle> {
+                                                    return feature.javaClass.declaredMethods.mapNotNull { method ->
+                                                        if (method.genericReturnType.toString().contains(Toggle::class.java.canonicalName!!)) {
+                                                            method.invoke(feature) as Toggle
+                                                        } else {
+                                                            null
+                                                        }
+                                                    }
+                                                }
```

Comment:
I think you can remove all these. You can either leave it as `return `emptyList()` or maybe better, default `{}` implementations in the `FeatureTogglesInventory` interface.
`RealFeatureTogglesInventory` will implement them using `getAll`